systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterTest.java

```diff
@@ -152,4 +153,21 @@ String globalVariableJsonPathBuilder(String variable) {
                 .filter(event -> event.getInvolvedObject().getName().equals(resourceName))
                 .collect(Collectors.toList());
     }
+
+    public void sendMessages(String messages, String clusterName, String topic) {
+        LOGGER.info("Sending messages");
+        String command = "echo -e \"" + messages + "\" | sh bin/kafka-console-producer.sh --broker-list " +
+                clusterName + "-kafka:9092 --topic " + topic + " & sleep 20";
+        kubeClient.exec(kafkaPodName(clusterName, 1), "/bin/bash", "-c", command);
+    }
+
+    public List<String> consumeMessages(String clusterName, String topic) {
+
+        LOGGER.info("Consuming messages");
+        String output = kubeClient.exec(kafkaPodName(clusterName, 1), "/bin/bash", "-c",
+                    "/opt/kafka/bin/kafka-console-consumer.sh --bootstrap-server " + clusterName +
```

Comment:
out of curiosity, why are you using the absolute path for the `kafka-console-consumer.sh` here while you used a relative path for the `kafka-console-producer.sh` ?